### PR TITLE
MongoDb ODM v2 Support

### LINKED ISSUE #218: Upgrade to support mongo-odm v2
MongoDB Subscriber has to change to support mongo-odm v2

### PR DESCRIPTION
**Pull request by markitosgv**
Upgrade to support Doctrine ODM 2.0

Closes #218

Increase php minor version to 7.2 because of mongo-dom v2 requirements

### CODE REVIEW

**Comment by markitosgv** [comment]
@garak change to this PR

File: composer.json [hunk 1]
```diff
@@ -17,12 +17,12 @@
...
     ],
     "require": {
-        "php": ">=7.1"
+        "php": ">=7.2"
     },
     "require-dev": {
         "symfony/event-dispatcher": "^3.4 || ^4.0",
         "doctrine/orm": "~2.4",
-        "doctrine/mongodb-odm": "~1.0@beta",
+        "doctrine/mongodb-odm": "^2.0.0-beta1",
```
Comment (garak) [inline composer.json]: We can target `^2.0.0-rc1`

File: composer.json [hunk 2]
```diff
@@ -17,12 +17,12 @@
         }
     ],
     "require": {
-        "php": ">=7.1"
+        "php": ">=7.2"
```
Comment (garak) [inline composer.json]: Because of this change, travis configuration needs to be updated

Comment (markitosgv) [reply composer.json]: I see php 7.2 is configured in .travis.yml on 2.0 branch

Comment (garak) [reply composer.json]: Yes, but you need to exclude 7.1

File: src/Knp/Component/Pager/Event/Subscriber/Paginate/Doctrine/ODM/MongoDB/QuerySubscriber.php [hunk 3]
```diff
@@ -42,7 +44,6 @@ public function items(ItemsEvent $event): void
             $event->stopPropagation();
         }
     }
-
```
Comment (garak) [inline src/Knp/Component/Pager/Event/Subscriber/Paginate/Doctrine/ODM/MongoDB/QuerySubscriber.php]: Please restore this blank linke

**Comment by garak** [comment]
Thank you @markitosgv

**Pull request merged**